http4k-core/src/main/kotlin/org/http4k/core/cookie/Cookie.kt

```diff
@@ -15,14 +16,20 @@ data class Cookie(val name: String, val value: String,
                   val domain: String? = null,
                   val path: String? = null,
                   val secure: Boolean = false,
-                  val httpOnly: Boolean = false) {
+                  val httpOnly: Boolean = false,
+                  val sameSite: SameSite? = null) {
 
     fun domain(domain: String) = copy(domain = domain)
     fun maxAge(seconds: Long) = copy(maxAge = seconds)
     fun path(path: String) = copy(path = path)
     fun secure() = copy(secure = true)
     fun httpOnly() = copy(httpOnly = true)
     fun expires(date: LocalDateTime): Cookie = copy(expires = date)
+    fun sameSite(sameSite: SameSite) = copy(sameSite = sameSite)
+
+    enum class SameSite {
```

Comment:
Don't think this shoud be an inner class of the Cookie - let's bump it up to the top level (we are in the "cookie" package :) )

Reply:
I'll move this after merging it.